include/experimental/__p0009_bits/layout_stride.hpp

```diff
@@ -651,15 +651,15 @@ constexpr void validate_strides(with_rank<N>, Layout, const Extents& ext, const
 
   constexpr auto is_left = std::is_same<Layout, layout_left>::value;
 
-  typename Extents::index_type stride = 1;
+  typename Extents::index_type stride_ = 1;
```

Comment:
Should we consider avoiding use of trailing underscore for variables that aren't nonpublic data members of the class?

```suggestion
  typename Extents::index_type the_stride = 1;
```